templates/data/mongodb/support.html

```diff
@@ -0,0 +1,429 @@
+                        alt="Kubeflow logo",
+                        width="308",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="is-muted" />
+            <p>
+              <a href="https://charmed-kubeflow.io/">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/a8fbddc8-apache-spark-logo.png",
+                        alt="Apache Spark logo",
+                        width="241",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="is-muted" />
+            <p>
+              <a href="/data/spark">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/24b77e42-mlflow-logo.png",
+                        alt="MLflow logo",
+                        width="222",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="is-muted" />
+            <p>
+              <a href="https://ubuntu.com/ai/mlflow">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/f6cce0b9-opensearch-logo-2.png",
+                        alt="OpenSearch logo",
+                        width="331",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="is-muted" />
+            <p>
+              <a href="/data/opensearch">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/9899746b-kafka-logo.png",
+                        alt="Kafka logo",
+                        width="239",
+                        height="312",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-logo-section__logo"}) | safe
+            }}
+            <hr class="is-muted" />
+            <p>
+              <a href="/data/kafka">Learn more&nbsp;&rsaquo;</a>
+            </p>
+          </div>
+          <div class="p-logo-section__item">
+            {{ image(url="https://assets.ubuntu.com/v1/662c72e2-mongodb.png",
```

Comment:
MongoDB logo looks like it's just falling back to it's alt text even though it isn't. Left a [comment](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl?node-id=3-8107#911214640) on the figma